engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -260,7 +260,6 @@ <h4 translate>Content Counts</h4>
   <section class="well" ng-if="permitted('edit_products', product) && !product.redhat">
     <h5 translate ng-show="repository.content_type == 'puppet'">Upload Puppet Module</h5>
     <h5 translate ng-show="repository.content_type == 'yum'">Upload Package</h5>
-    <h5 translate ng-show="repository.content_type == 'docker'">Upload Docker Image</h5>
```

Comment:
Spoke with @jlsherrill after PR'ing this, I need to adding another conditional to the `ng-if` statement to not show the entire section.